app/(auth)/actions.ts

```diff
@@ -24,7 +24,7 @@ export async function signin(formData: FormData) {
   }
 
   revalidatePath("/", "layout");
-  redirect("/");
+  redirect("/settings");
```

Comment:
I think we keep it to redirect to home page even if they signed in for now. Cause our settings page does not have much info yet

Reply:
Ok sounds good